decidim-system/app/views/decidim/system/devise/passwords/edit.html.erb

```diff
@@ -1,22 +1,19 @@
-<h2>Change your password</h2>
+<h2 class="h3">Change your password</h2>
 
 <%= decidim_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f| %>
   <%= render "devise/shared/error_messages", resource: %>
   <%= f.hidden_field :reset_password_token %>
 
-  <div class="field">
+  <div class="form__wrapper">
     <% if @minimum_password_length %>
       <em>(<%= @minimum_password_length %> characters minimum)</em><br>
```

Comment:
This string should be extracted from here. (not in scope of this PR)

Reply:
@alecslupu isn't this a class name? What i18n string could be extracted to be added here?

Reply:
It's on the line number 9, "(<%= @minimum_password_length %> characters minimum)"